tasks/components/layout/TaskDetailView.tsx

```diff
@@ -0,0 +1,157 @@
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import type { TaskDTO } from '../../mutations/room_mutations'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { tw } from '@helpwave/common/twind/index'
+import { ToggleableInput } from '../user_input/ToggleableInput'
+import Close from '@helpwave/common/icons/Close'
+import { Textarea } from '../user_input/Textarea'
+import { Select } from '../user_input/Select'
+import { TaskStatusSelect } from '../user_input/TaskStatusSelect'
+import { Button } from '../Button'
+import { SubtaskView } from '../SubtaskView'
+
+type TaskDetailViewTranslation = {
+  close: string,
+  notes: string,
+  subtasks: string,
+  assignee: string,
+  dueDate: string,
+  status: string,
+  visibility: string,
+  creationTime: string,
+  private: string,
+  public: string,
+  create: string,
+  update: string
+}
+
+const defaultTaskDetailViewTranslation = {
+  en: {
+    close: 'Close',
+    notes: 'Notes',
+    subtasks: 'Subtasks',
+    assignee: 'Assignee',
+    dueDate: 'Due-Date',
+    status: 'Status',
+    visibility: 'Visibility',
+    creationTime: 'Creation Time',
+    private: 'private',
+    public: 'public',
+    create: 'Create',
+    update: 'Update'
+  },
+  de: {
+    close: 'Schließen',
+    notes: 'Notizen',
+    subtasks: 'Unteraufgaben',
+    assignee: 'Veranwortlich',
+    dueDate: 'Fälligkeits-Datum',
+    status: 'Status',
+    visibility: 'Sichbarkeit',
+    creationTime: 'Erstell Zeit',
+    private: 'privat',
+    public: 'öffentlich',
+    create: 'Hinzufügen',
+    update: 'Ändern'
+  }
+}
+
+type TaskDetailViewProps = {
+  task: TaskDTO,
+  onChange: (task: TaskDTO) => void,
+  onClose: () => void,
+  onFinishClick: () => void
+}
+
+export const TaskDetailView = ({
+  language,
+  task,
+  onChange,
+  onClose,
+  onFinishClick
+}: PropsWithLanguage<TaskDetailViewTranslation, TaskDetailViewProps>) => {
+  const translation = useTranslation(language, defaultTaskDetailViewTranslation)
+  return (
+    <div className={tw('flex flex-col h-full')}>
+      <div className={tw('flex flex-row justify-between items-center')}>
+        <ToggleableInput
+          id={task.id}
+          value={task.name}
+          onChange={name => onChange({ ...task, name })}
+          labelClassName={tw('text-xl font-semibold')}
+        />
```

Comment:
Toggle Input should be in edit state by default
Also the edit icon is way too far away from the actual input